.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,11 @@
-<!-- Please take a look at our Contributing documentation before submitting a Pull Request!
-https://rook.io/docs/rook/latest/Contributing/development-flow/
+<!-- Thank you for contributing to Rook! -->
 
-Thank you for contributing to Rook! -->
-
-**Description of your changes:**
+<!-- STEPS TO FOLLOW:
+  1. Add a description of the changes (frequently the same as the commit description)
+  2. Enter the issue number next to "Resolves #" below (if there is no tracking issue resolved, **remove that section**)
+  3. Review our Contributing documentation at https://rook.io/docs/rook/latest/Contributing/development-flow/
+  4. Review the checklist below
```

Comment:
How about adding one more step, step 5: saying something like steps 1-4 are required to get PR reviewed and merged?

Maybe this sounds little hard but we need something so that PR owners(including me) complete the steps for regularly otherwise I'm afraid it will be neglected after sometimes.

Reply:
Also, in some other upstream projects,  I have seen them follow the PR template seriously and they don't review the PR until the steps are marked.

Just raising the thoughts

Reply:
Good questions. I would agree it is really up to the maintainers to enforce the patterns before reviewing them if we really care about them. I don't think the 5th bullet would help because if they didn't read the first four, they won't read the fifth one either. :)